pkgs/development/libraries/hare-json/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
+, fetchFromSourcehut
+, hare
+, unstableGitUpdater
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hare-json";
+  version = "unstable-2023-02-25";
+
+  src = fetchFromSourcehut {
+    owner = "~sircmpwn";
+    repo = pname;
+    rev = "57b96f905e2b99f4b4d5f273a22dc3bc7c796513";
+    hash = "sha256-q/e8iFawECDi2T72Nbp9Nk3vzmMdup9vgyS3vhYUFEM=";
```

Comment:
up to 99ae40eacc19253495949301000372adf8c3f504 works on current hare version in nixpkgs, you can bump it